common/src/main/scala/models/Notification.scala

```diff
@@ -85,6 +86,121 @@ object ContentNotification {
   implicit val jf = Json.format[ContentNotification]
 }
 
+
+case class FootballMatchStatusNotification(
+  id: UUID,
+  `type`: NotificationType = FootballMatchStatus,
+  title: String,
+  message: String,
+  thumbnailUrl: Option[URI] = None,
+  sender: String,
+  awayTeamName: String,
+  awayTeamScore: Int,
+  awayTeamMessage: String,
+  awayTeamId: String,
+  homeTeamName: String,
+  homeTeamScore: Int,
+  homeTeamMessage: String,
+  homeTeamId: String,
+  competitionName: Option[String],
+  venue: Option[String],
+  matchId: String,
+  mapiUrl: URI,
+  importance: Importance,
+  topic: Set[Topic],
+  phase: String,
+  eventId: String,
+  debug: Boolean
+) extends Notification {
+  override def withTopics(topics: Set[Topic]): Notification = copy(topic = topics)
+}
+
+object FootballMatchStatusNotification {
+  import JsonUtils._
+
+  implicit val jf = new Format[FootballMatchStatusNotification] {
+    override def reads(json: JsValue): JsResult[FootballMatchStatusNotification] = {
```

Comment:
I suppose something is holding you back from using Play's macro but I can't spot it!